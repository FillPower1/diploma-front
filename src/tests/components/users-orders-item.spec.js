import React from 'react'
import { shallow } from 'enzyme'
import UsersOrdersItems from '../../components/users-orders-item'

describe('Users Orders Items', () => {
	const props = {}

	describe('UsersOrdersItems render', () => {

		const nextProps = {
			...props,
			title: "some text",
			count: 5
		}

		const usersOrdersItems = shallow(<UsersOrdersItems {...nextProps} />)

		it('renders properly', () => {
			expect(usersOrdersItems).toMatchSnapshot()
		})

		it('render UsersOrdersItems title', () => {
			expect(usersOrdersItems.find('.order__title').text()).toEqual(nextProps.title)
		})

		it('render UsersOrdersItems count', () => {
			expect(usersOrdersItems.find('.order__count').text()).toEqual(`Количество: ${nextProps.count}`)
		})
	})
})