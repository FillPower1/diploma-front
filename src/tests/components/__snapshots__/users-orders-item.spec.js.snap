// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Users Orders Items UsersOrdersItems render renders properly 1`] = `ShallowWrapper {}`;
